lib/sensors/MoistureSensor.cpp

```diff
@@ -0,0 +1,21 @@
+#include "mbed.h"
+#include <I2C.h>
```

Comment:
what is this header? `#include "mbed.h"` should pull in everything needed for i2c